internal/pkg/service/common/configmap/dump_test.go

```diff
@@ -17,7 +17,7 @@ func TestDumpFlat_Empty(t *testing.T) {
 	require.NoError(t, err)
 	assert.Equal(t, strings.TrimSpace(`
 {
-  "address": "invalid IP",
+  "address": "",
```

Comment:
`<type>.String()` returns "invalid IP", `<type>.MarshalText()` returns `""`, now it is consistent, see other fields in the test.